app/models/user.rb

```diff
@@ -343,6 +343,10 @@ def pro?
     end
   end
 
+  def vomitted?
```

Comment:
Maybe this should be `vomitted_on?` for clarity's sake?

Reply:
I agree :+1: